src/main/resources/model/bibframe_authority.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "bibframe-authority",
+  "eventBodyJavaClass": "org.folio.search.domain.dto.BibframeAuthority",
+  "fields": {
+    "id": {
+      "index": "keyword"
+    },
+    "label": {
+      "index": "multilang"
```

Comment:
"multilang" index will not properly as there is no language-defining field in the model.
You could use "standard" to get full-text capabilities.